odata-protocol/11.4 Data Modification.md

```diff
@@ -696,7 +696,8 @@ request. The service MUST NOT treat an update request containing an
 
 A `PUT` or `PATCH` request MUST NOT be treated as an update if an
 [`If-None-Match`](#HeaderIfNoneMatch) header is specified with a value
-of `*`.
+of `*` or the `@etag` control information [#OData-JSON#ControlInformationetagodataetag]
+is specified with an empty string as value.
```

Comment:
Better add this after https://github.com/oasis-tcs/odata-specs/blob/ff14732045e46423b0752921e720d46b13827c5c/odata-protocol/11.4%20Data%20Modification.md?plain=1#L1168

and use the same format-agnostic formulation.

So far we do not mention `@etag` in Protocol.